packages/expo/bin/cli.ts

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env node
+import arg from 'arg';
+import chalk from 'chalk';
+
+const defaultCmd = 'config';
+
+export type Command = (argv?: string[]) => void;
+
+const commands: { [command: string]: () => Promise<Command> } = {
+  // Add a new command here
+  config: () => import('../cli/config').then((i) => i.expoConfig),
+};
+
+const args = arg(
+  {
+    // Types
+    '--version': Boolean,
+    '--help': Boolean,
+
+    // Aliases
+    '-v': '--version',
+    '-h': '--help',
+  },
+  {
+    permissive: true,
+  }
+);
+
+if (args['--version']) {
+  // Version is inlined into the file using taskr build pipeline
+  console.log(`expo v${process.env.__EXPO_VERSION}`);
+  process.exit(0);
+}
+
+// Check if we are running `npx expo <subcommand>` or `npx expo`
+const targetCmd = Boolean(commands[args._[0]]);
+
+// Handle `--help` flag
+if (!targetCmd && args['--help']) {
+  console.log(chalk`
+    {bold Usage}
+      {bold $} npx expo <command>
+
+    {bold Available commands}
+      ${Object.keys(commands).join(', ')}
```

Comment:
We might want to think about adding a command + 1 line description, which might help users determine the right commands. Just like what we currently have for Expo CLI and EAS CLI.

![image](https://user-images.githubusercontent.com/1203991/148640774-606adef0-c26d-4c90-9f70-c74da1a07990.png)

Reply:
Can maybe add after we have more than one command. Currently we lazy load all of the command contents after a command is invoked, so we'd need to load all command help descriptions ahead of time.